app/services/npq/reject_application.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module NPQ
+  class RejectApplication
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+    include ActiveModel::Validations::Callbacks
+
+    attribute :npq_application
+
+    validates :npq_application, presence: { message: I18n.t("npq_application.missing_npq_application") }
+    validate :not_already_rejected
+    validate :cannot_change_from_accepted
+
+    def call
+      return self unless valid?
+
+      npq_application.update!(lead_provider_approval_status: "rejected")
+      npq_application
+    end
+
+  private
+
+    def not_already_rejected
```

Comment:
I think this validation is superfluous. If the Application is already rejected no need to return and error just skip the 
` npq_application.update!(lead_provider_approval_status: "rejected")`